study-builder/studies/singular/study.conf

```diff
@@ -0,0 +1,104 @@
+{
+  "governance": {
+    # This rule here is only used in the normal registration flow, not the add-child flow.
+    "shouldCreateGovernedUserExpr": """
+      user.studies["singular"].forms["CONSENT_PARENTAL"].hasInstance()
+      && user.studies["singular"].forms["CONSENT_PARENTAL"].isStatus("COMPLETE")
+    """,
+    # These rules apply to both normal registration and add-child flows, so check both.
+    "ageOfMajorityRules": [
```

Comment:
@ssettipalli AFAIK this is the first **multi-governed** study with configured Age Up process. And the difference here is that PREQUAL is owned by the operator. In order to access that from PEX in the AgeUpService context we need to save this operator who enrolled the user.

Reply:
Existing studies like Osteo and pancan refer to governed users prequal during AgeUp check and also operators can add multiple child's (governed users) . 
For ex:-
user.studies["CMI-OSTEO"].forms["PREQUAL"].questions["PREQUAL_SELF_DESCRIBE"].answers.hasOption("CHILD_DIAGNOSED")
          && user.studies["CMI-OSTEO"].forms["PREQUAL"].questions["CHILD_COUNTRY"].answers.hasOption("PR")

Reply:
For OSTEO there is a reference to PREQUAL, but there is no multi-user.
In PANCAN there is multi-governed enabled but AOM rules check if there is PREQUAL for existing user and uses it, otherwise check country/state on ADD_CHILD activity.
We do not have country/state questions on ADD_PARTICIPANT activity. So we had to use operator’s PREQUAL. However, there was a flaw: in order to make ‘operator’ work in PEX we need to pass it to interpreter. But we don’t know who was the operator. We could find this from user_governance but according to DB schema there could be more than one operator! So we put operator into age up candidate when it is being created - we know who’s the operator at this point of time

Reply:
yeah, probably cleaner to populate operator in there when candidate is created